types/office-js/index.d.ts

```diff
@@ -973,15 +973,25 @@ declare namespace Office {
     }
 
     /**
-     * Provides information about what Requirement Sets are supported in current environment.
+     * Provides information about which Requirement Sets are supported in the current environment.
      */
     interface RequirementSetSupport {
         /**
         * Check if the specified requirement set is supported by the host Office application.
         * @param name - Set name; e.g., "MatrixBindings".
-        * @param minVersion - The minimum required version; e.g., "1.4". Note: String type is recommended data type for this parameter. The use of number type is deprecated and will not be compatible with recent requirement sets.
+        * @param minVersion - The minimum required version (e.g., "1.4").
```

Comment:
For the `@param name` description
- would it perhaps be more helpful to say `e.g., "ExcelApi"` instead of `e.g., "MatrixBindings"` -- since most folks won't even recognize "MatrixBindings" as a requirement set name?
- suggest you use the same structure/format for this line as is used in the `@param minVersion` line: 
    `@param name - Set name (e.g., "ExcelApi").`